retail/interactive-tutorial/RetailProducts.Samples/SetInventorySample.cs

```diff
@@ -0,0 +1,126 @@
+﻿// Copyright 2021 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START retail_set_inventory]
+
+using Google.Cloud.Retail.V2;
+using Google.Protobuf.WellKnownTypes;
+using System;
+using System.Threading;
+
+/// <summary>
+/// The set inventory sample class.
+/// </summary>
+public class SetInventorySample
+{
+    /// <summary>
+    /// Get the retail product with inventory info.
+    /// </summary>
+    /// <param name="product">The actual product.</param>
+    /// <returns>Retail product with updated inventory info.</returns>
+    private static Product SetInventoryForProduct(Product product)
+    {
+        string[] placeIds = { "store1", "store2" };
+
+        product.FulfillmentInfo[0].Type = "pickup-in-store";
+        product.FulfillmentInfo[0].PlaceIds.Clear();
+        product.FulfillmentInfo[0].PlaceIds.AddRange(placeIds);
```

Comment:
You can only access the element at 0 if the product had a fulfillment already.
Maybe consider showing how to deal with one that doesn't has a fullfillment?

```suggestion
        product.FulfillmentInfo.Clear();
        product.FulfillmentInfo.Add(new ....);
```